specification.md

```diff
@@ -1855,6 +1874,54 @@ for this connection.
 This field is optional: if omitted the client SHOULD NOT use a client-side certificate.
 This field can be used to perform a client certificate revocation/rotation.
 
+##### OpAMPConnectionSettings.heartbeat_interval_seconds
+
+If the ReportsHeartbeat capability is true, the Client MUST use the offered heartbeat
+interval to periodically send an AgentToServer message. If the capability is true
+and the Server sets heartbeat_interval_seconds to 0, Agent heartbeats should be disabled.
```

Comment:
```suggestion
and the Server sets heartbeat_interval_seconds to 0, Agent heartbeats MUST be disabled.
```
I think if we require that the agent respects the interval set by the server, it needs to respect disabling heartbeats as well.